### PR TITLE
Date.issue#160

### PR DESCRIPTION
**Pull request by emli**

### CODE REVIEW

File: download/download.go [hunk 1]
```diff
@@ -238,12 +244,14 @@ func formatItem(info map[string]string, repo, name string) ListItem {
...
 		Architecture: strings.ToUpper(info["arch"]),
 		Description:  info["Description"],
+		Timestamp:    timestamp,
 	}
 	item.Size, _ = strconv.Atoi(info["size"])
 
 	if repo == "apt" {
 		item.Version = info["Version"]
 		item.Architecture = info["Architecture"]
+		item.Size, _ = strconv.Atoi(info["Size"])
```
Comment (Dilshat) [inline download/download.go]: do we have size only for deb packages? why not add this field to all artifacts?

File: download/download.go [hunk 2]
```diff
@@ -22,6 +22,8 @@ type ListItem struct {
 	ID           string            `json:"id"`
 	Hash         hashsums          `json:"hash"`
 	Size         int               `json:"size"`
+	Date         string            `json:"upload-date-formated"`
```
Comment (crioto) [inline download/download.go]: Fix `formatted` type

File: download/download.go (hunk 1 above)
Comment (emli) [reply download/download.go]: We have size field for template,raw files. We found bug only for apt

File: download/download.go (hunk 2 above)
Comment (emli) [reply download/download.go]: fixed

File: download/download.go (hunk 1 above)
Comment (Dilshat) [reply download/download.go]: ok

**Pull request merged**